testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -1279,6 +1281,80 @@ public void testHolderOfKeyEnforceExecutor() throws Exception {
         }
     }
 
+
+    @Test
+    public void testExtendedClientPolicyIntefacesForClientRegistrationPolicyMigration() throws ClientRegistrationException, ClientPolicyException {
+        String policyName = "MyPolicy";
+        String clientName = "ByAdmin-App" + KeycloakModelUtils.generateId().substring(0, 7);
+        String executorName = "TestRaiseExeptionExecutor";
+
+        createPolicy(policyName, DefaultClientPolicyProviderFactory.PROVIDER_ID, null, null, null);
+        logger.info("... Created Policy : " + policyName);
+
+        createCondition("AnyClientConditionFactory", AnyClientConditionFactory.PROVIDER_ID, null, (ComponentRepresentation provider) -> {
+        });
+        registerCondition("AnyClientConditionFactory", policyName);
+        logger.info("... Registered Condition : AnyClientConditionFactory");
+
+        createExecutor(executorName, TestRaiseExeptionExecutorFactory.PROVIDER_ID, null, (ComponentRepresentation provider) -> {
+            provider.getConfig().put(TestRaiseExeptionExecutorFactory.TARGET_CP_EVENTS, new ArrayList<>(Arrays.asList(ClientPolicyEvent.REGISTERED.toString())));
+        });
+        registerExecutor(executorName, policyName);
+        logger.info("... Registered Executor : " + executorName);
+
+        String clientId = null;
+        try {
+            try {
+                createClientByAdmin(clientName, (ClientRepresentation clientRep) -> {
+                });
+                fail();
+            } catch (ClientPolicyException cpe) {
+                assertEquals(ClientPolicyEvent.REGISTERED.toString(), cpe.getError());
+            }
+
+            updateExecutor(executorName, (ComponentRepresentation provider) -> {
```

Comment:
Pattern
```
updateExecutor(executorName, (ComponentRepresentation provider) -> {
                provider.getConfig().put(TestRaiseExeptionExecutorFactory.TARGET_CP_EVENTS, new ArrayList<>());
            });
```         
is used in multiple places

Can we create private helper method:
```
void updateTargetCPEvents(String executorName, List<ClientPolicyEvent> events) {
  updateExecutor(executorName, (ComponentRepresentation provider) -> {
                provider.getConfig().put(TestRaiseExeptionExecutorFactory.TARGET_CP_EVENTS, events.stream().map(ClientPolicyEvent::toString()).collect(Collectors.toList()));
  });
}
```

So that

```
updateExecutor(executorName, (ComponentRepresentation provider) -> {
                provider.getConfig().put(TestRaiseExeptionExecutorFactory.TARGET_CP_EVENTS, new ArrayList<>());
            });
```         

becomes updateTargetCPEvents(executorName,  Arrays.asList(ClientPolicyEvent.UPDATED));

Reply:
@valb3r I see, I'll fix it as you suggested.